MediaAdmin/FileAlternatives/Strategy/ImageStrategy.php

```diff
@@ -148,9 +148,12 @@ public function deleteAlternatives(MediaInterface $media)
      */
     public function overrideAlternative(MediaInterface $media, $newFilePath, $formatName)
     {
-        $alternativeName = $this->getAlternativeName($formatName, $media->getFilesystemName());
+        $alternativeName = $media->getAlternative($formatName);
         $this->deleteFile($alternativeName);
-        $this->mediaStorageManager->uploadFile($alternativeName, $newFilePath);
+        $newFilename = pathinfo($newFilePath, PATHINFO_BASENAME);
+        $newAlternativeName = $this->getAlternativeName($formatName, $newFilename);
```

Comment:
I'm not sure with this. It works for now as the strategies provide a unique temporary filename. But if in the future a refacto on the resize changes that point, or if some stuff is inserted between the resize and that function breaking the unicity of the file name, it will produce errors on this code.
Maybe the file name should be regenerated here to be sure of the unicity of the name, whatever come from the parameters.